package.json

```diff
@@ -10,7 +10,8 @@
     "docker:stop": "./bin/stop-docker.sh",
     "e2e-dev": "API_ENV=dev jest --rootDir ./tests/e2e ",
     "e2e": "API_ENV=prod jest --rootDir ./tests/e2e ",
-    "local": "DB_PORT=3002 npm run docker:start && DB_PORT=3002 NODE_ENV=development node app.js",
+    "local": "npm run docker:start && npm run local:api",
+    "local:api": "DB_PORT=3002 DB_HOST='127.0.0.1' NODE_ENV=development node app.js",
```

Comment:
if you pass `DB_PORT=3002` here for the api, the docker script will need it too or it will run the DB on 3306.

makes the previous line 
```
 "local": "DB_PORT=3002 npm run docker:start && npm run local:api",
```

Reply:
its going on the right port in my testing. I removed due to redundancy:

"docker:start": "DB_PORT=3002 ./bin/start-docker.sh",